app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -2066,6 +2099,7 @@ class BrowserTabFragment :
             when (configuration) {
                 is HomePanelCta -> showHomeCta(configuration, favorites)
                 is DaxBubbleCta -> showDaxCta(configuration)
+                is BubbleCta -> showBubleCta(configuration)
```

Comment:
We need to send a pixel each time we show the favorite onboarding cta.
Using `DaxBubbleCta` (`DaxCta` interface) will not work, it will only send a pixel the first time (part of the dax onboarding logic).
I created a new cta that doesn't implement `DaxCta`, however the rendering logic is the same one. It may be not the best solution, but I wanted to avoid any refactor around CTAs inside the PR.